internal/controller/operatorconfigmap_controller.go

```diff
@@ -890,3 +784,79 @@ func (c *OperatorConfigMapReconciler) deleteDelegatedCSI() error {
 	}
 	return nil
 }
+
+func (c *OperatorConfigMapReconciler) reconcileSubscriptionLabelForWebhook() error {
+	sub, err := c.getSubscriptionByPackageName("ocs-client-operator")
+	if err != nil {
+		return err
+	}
+	if utils.AddLabel(sub, subscriptionLabelKey, subscriptionLabelValue) {
+		if err := c.update(sub); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *OperatorConfigMapReconciler) reconcileSubscriptions() error {
+	pkgNames := [][2]string{
+		// d/s and u/s names of packages for fallback
+		{"ocs-client-operator", ""},
+		{"odf-csi-addons-operator", "csi-addons"},
+		{"cephcsi-operator", "ceph-csi-operator"},
+		{"recipe", ""},
```

Comment:
Why to we need the upstream/downstream distinction? I think we can just have a single list which include all names